### ISSUE TITLE
Make sure cut outs do not overlap headlines on fronts

### ISSUE THREAD

**Issue by paperboyo** [guardian/frontend#8935]
[for a possible quick fix, look for __bold__ text below]

Hello,

I just wanted to log that one. Currently, it's impossible to make sure that a cut out won't overlap headline text on fronts when preparing an image. We should make sure that we do everything possible so that does not happen. All cut outs are done to the same template. Here is a front populated with them (thanks to @Pearson82): http://m.code.dev-theguardian.com/cutouttest

![cut out test_all bps](https://cloud.githubusercontent.com/assets/6032869/7247152/21a8fb18-e7fe-11e4-848e-8d1d784b5e96.jpg)

Every now and then we get a report of a clash, like this one from today (reported by Subhajit B.):

![img_0891](https://cloud.githubusercontent.com/assets/6032869/7247194/9ec3af8a-e7fe-11e4-8b53-6040b253be66.jpg)

The easiest/fastest fix would be to make cut outs appear __behind__ text. But the careful adjustment of image placement/scale on differently sized tiles may be necessary, along with trying to sort out hyphenation (related to https://github.com/guardian/frontend/issues/8784) before we can sort all this out by using ``<image>`` part of ``CSS shapes`` spec (currently only in Chrome, Safari?).

Regards
Mateusz

**Comment by gidsg** [comment]
A similar issue can happen with cutouts on articles, e.g. http://www.theguardian.com/culture/2015/apr/12/sex-train-parents-pornography-eva-wiseman at 980px width the cut-out overlaps the secondary nav making the last 2 links un-clickable
![screen shot 2015-04-21 at 09 48 33](https://cloud.githubusercontent.com/assets/1764158/7248600/d3c237a8-e80b-11e4-8762-88787d36fb7a.png)
Do you want me to raise this as a separate Issue? cc @michaelwmcnamara

**Comment by paperboyo** [comment]
Whoa! The inability to click both stage and classical links above is really severe! :( The template these are made to has a fixed height (and width) and the top ~36px is used only (rarely) for subjects with extensive hair (still we are waiting for our Hendrix...). I very much like a slight overlap of top of the hair and secondary nav but the inability to click links (or to read all text) is unacceptable.

Regards
Mateusz

**Comment by JustinPinner** [comment]
The fix for @gidsg's bug is now out. You can happily click on Eva Wiseman's head and still get to the classical sub-section. The closed PR is here: https://github.com/guardian/frontend/pull/9017

**Comment by johnduffell** [comment]
I'm sure this is fixed, if you have too many links in the sec nav it should shrink the cutout slightly to avoid a crash

**Comment by paperboyo** [comment]
@gidsg's https://github.com/guardian/frontend/issues/8935#issuecomment-94707577 is fixed. Original issue is still not fixed as per huge complicated image above...
